docs/vehicle_model.md

```diff
@@ -9,16 +9,57 @@ One of the most controversial subjects of any competition simulator is the vehic
 * **A third-party, open-source model would be ideal**. This way, not even the developers of the simulation (Formula Student Team Delft) would have an edge over other teams. Everyone has access to the same code and has had no experience working with it or been involved in developing it.
 
 
-The Unreal Engine repository contains (as a third party library) the code for [PhysXVehicles](https://docs.nvidia.com/gameworks/content/gameworkslibrary/physx/guide/Manual/Vehicles.html) that was developed by Nvidia. This seemed like the perfect solution for our simulator, given that it complies with all the criteria of our design philosophy above. Airsim simply interacts with the PhysXCar API in [these](https://github.com/FS-Online/Formula-Student-Driverless-Simulator/tree/93fa784106c70e53e973e6d755ab5430feeef9da/UE4Project/Plugins/AirSim/Source/Vehicles/Car) files, which have configurable parameters exposed in the uassets files [here](https://github.com/FS-Online/Formula-Student-Driverless-Simulator/tree/master/UE4Project/Plugins/AirSim/Content/VehicleAdv/SUV). 
+Unreal Engine provides the [PhysXVehicles](https://docs.nvidia.com/gameworks/content/gameworkslibrary/physx/guide/Manual/Vehicles.html) that was developed by Nvidia. 
+This seemed like the perfect solution for our simulator, given that it complies with all the criteria of our design philosophy above. 
+Airsim simply interacts with the PhysXCar API in [these](https://github.com/FS-Online/Formula-Student-Driverless-Simulator/tree/master/UE4Project/Plugins/AirSim/Source/Vehicles/Car) files.
 
-These are the current (high-level) vehicle parameters (you will be notified if anything changes):
+## Vehicle pawn center
+
+Just like everything inside the simulated world, the vehicle has a position within the world (x, y, z).
+Relative to this point the rest of the vehicle is constructed.
+The collision box, the center of gravity and the sensor positions all are relative to this center point.
+
+The center position is located at the bottom of the car and on the horizontal plane in the geometric center.
+When car is stationary the height offset between the world track and the pawn position is 0.
+In below image the blue dot represents the location of the pawn center:
+
+![vehicle layout 4 angles](images/vehicle-layout.png)
```

Comment:
This image does not show up in preview and the link seems to be broken

Reply:
What previewer are you using? When looking here it looks fine https://github.com/FS-Online/Formula-Student-Driverless-Simulator/blob/center-of-gravity/docs/vehicle_model.md